interface.html

```diff
@@ -1051,7 +1051,24 @@ <h3><small>Reviewer Notes</small></h3>
                     <p class="help-block label-helper"><small>Optional</small></p>
                   </div>
                   <div class="col-sm-8">
-                    <textarea name="fl-store-revNotes" class="form-control" id="fl-store-revNotes" rows="10"></textarea>
+                    <textarea name="fl-store-revNotes" class="form-control" id="fl-store-revNotes" rows="10" required>
```

Comment:
@ibroom why did you add the `required`?

Reply:
The field should be mandatory.

Reply:
I understand but changing it on the code will break enterprise submissions (since the field is not required). I believe the "required" functionality will have to be dealt with on the JS code of the component, @hcarneiro correct me if I'm wrong. 

@ibroom this is also to point out that — once again — we need change requests via tickets, not non-developers writing code that we have to review.

Reply:
@ibroom I agree with @squallstar. We want to take code review quite seriously, and it's really best to have the changes made by developers who know how to best implement the features/changes. Otherwise time is wasted through reviewing sub-par code and requesting for justifications/changes because the person who implements the change becomes responsible for the code.

Please submit change requests via tickets.

Reply:
I did :D https://github.com/Fliplet/fliplet-studio/issues/2387